app/print.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABC, abstractmethod
+
+from app.book import Book
+
+
+class Print(ABC):
+    @abstractmethod
+    def print(self, book: "Book") -> None:
+        pass
+
+
+class ConsolePrint(Print):
+    def print(self, book: "Book") -> None:
+        print(f"Printing the book: {book.title}...")
+        print(book.content)
+
+
+class ReversePrint(Print):
```

Comment:
So, it was originally named `Display`. In that case, `DisplayReverse` would be a more descriptive name.